x/group/client/cli/tx.go

```diff
@@ -284,6 +286,106 @@ func MsgUpdateGroupMetadataCmd() *cobra.Command {
 	return cmd
 }
 
+// MsgCreateGroupWithPolicyCmd creates a CLI command for Msg/CreateGroupWithPolicy.
+func MsgCreateGroupWithPolicyCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "create-group-with-policy [admin] [group-metadata] [group-policy-metadata] [members-json-file] [decision-policy]",
+		Short: "Create a group with policy which is an aggregation " +
+			"of member accounts with associated weights, " +
+			"an administrator account and a decision policy. Note, the '--from' flag is " +
+			"ignored as it is implied from [admin].",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Create a group with policy which is an aggregation of member accounts with associated weights,
+an administrator account and decision policy. Note, the '--from' flag is ignored as it is implied from [admin].
+Members accounts can be given through a members JSON file that contains an array of members.
+If group-policy-as-admin flag is set to true, the admin of the newly created group and group policy is set with the group policy address itself.
+
+Example:
+$ %s tx group create-group-with-policy [admin] [group-metadata] [group-policy-metadata] [members-json-file] \
+'{"@type":"/cosmos.group.v1beta1.ThresholdDecisionPolicy", "threshold":"1", "timeout":"1s"}'
+
+where members.json contains:
+
+{
+	"members": [
+		{
+			"address": "addr1",
+			"weight": "1",
+			"metadata": "some metadata"
+		},
+		{
+			"address": "addr2",
+			"weight": "1",
+			"metadata": "some metadata"
+		}
+	]
+}
+`,
+				version.AppName,
+			),
+		),
+		Args: cobra.MinimumNArgs(5),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := cmd.Flags().Set(flags.FlagFrom, args[0])
+			if err != nil {
+				return err
+			}
+
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			groupPolicyAsAdmin, err := cmd.Flags().GetBool(FlagGroupPolicyAsAdmin)
```

Comment:
how about using `isGroupPolicyAsAdmin` as var name here?

Reply:
it's longer and doesn't bring more meaning, why should we use a different var name than the actual field name